extensions/odoo_theme/__init__.py

```diff
@@ -115,12 +115,12 @@ def _set_docname_as_class(_reference_node, _node_docname):
 def icon_role(name, rawtext, text, lineno, inliner, options=None, content=None):
     """ Implement an `icon` role for Odoo and Font Awesome icons. """
     for icon_class in text.split():
-        if not icon_class.startswith('fa-'):
+        if not (icon_class.startswith('fa-') or icon_class.startswith('oi-')):
             report_error = inliner.reporter.error(
                 f"'{icon_class}' is not a valid icon formatting class.", lineno=lineno
             )
             error_node = inliner.problematic(rawtext, rawtext, report_error)
             return [error_node], [report_error]
-    icon_html = f'<i class="{text}"></i>'
+    icon_html = f'<i class="oi {text}"></i>' if text.startswith('oi-') else f'<i class="{text}"></i>'
```

Comment:
Oh I see you're still using the `oi ` class prefix. I thought the point was to avoid using them (`fa` and `oi`) to ease the implementation (https://github.com/odoo/documentation/pull/8469#discussion_r1578065195).

It's better to use it tho, it's the right way to handle font icons.
But now it doesn't make sense not to do the same for Font Awesome and therefore modify the library as you've done [here](https://github.com/odoo/documentation/pull/8469) 😬.

The way you've done it here makes it possible to use the library correctly without complicating its implementation. Font Awesome should be implemented in the same way 🙂.

But I think this should be done in another PR targeting `15.0`. Could you handle it ?

Reply:
Thats a good point 👍 I will make a 15.0 [FIX] to update the FA implementation to use the same methodology that OI uses for the main class prefix.